doc/user-guide/push-notifications/Push-notifications-client-side.md

```diff
@@ -0,0 +1,158 @@
+## Using push notifications on client side
```

Comment:
I highly recommend to restructure the document:
* first provide the diagram and description of the XEP compliant PubSub based configuration.
* then provide the PubSub less diagram and describe the difference only.

Reply:
or at least avoid 2 big diagrams followed one by one right at the beginning of the document.